Redrawing Results Incorrect

## Description

When using the progress meter `tqdm` and using the parameter `leave`, meters should be redrawn in the location of a completed meter. There appears to be whitespace left behind when this redraw occurs. There has been a bug open in `tqdm` for a few years now. There is a comment by the author of that package that this bug is a problem in the upstream project, Jupyter / Jupyter Lab. I can verify that this bug still exists now. Screenshot is attached.

![image](https://user-images.githubusercontent.com/2775739/66721872-3d134200-edd5-11e9-8eb7-50c644631573.png)


## Reproduce

```
import time
import tqdm
for _ in tqdm.tqdm_notebook(range(10)):
    for _ in tqdm.tqdm_notebook(range(10000000), leave=False):
        pass
```

## Expected behavior

No extra accumulated white space.

## Context

- Operating System and version: macOS 10.15 (19A583)
- Browser and version: 77.0.3865.120 (Official Build) (64-bit)
- JupyterLab version: 1.1.4

<details><summary>Command Line Output</summary>
<pre>
jupyter lab
</pre>
</details>

<details><summary>Browser Output</summary>
<pre>
react-dom.development.js:21258 Download the React DevTools for a better development experience: https://fb.me/react-devtools
index.js:42 Starting application in workspace: "/lab"
default.js:56 Starting WebSocket: ws://localhost:8888/api/kernels/0a0fe54b-ceea-4c29-a433-fb1dbedb5d40
default.js:56 Starting WebSocket: ws://localhost:8888/api/kernels/0a0fe54b-ceea-4c29-a433-fb1dbedb5d40
default.js:1496 Kernel: connected (0a0fe54b-ceea-4c29-a433-fb1dbedb5d40)
default.js:1496 Kernel: connected (0a0fe54b-ceea-4c29-a433-fb1dbedb5d40)
default.js:56 Starting WebSocket: ws://localhost:8888/api/kernels/6d8f9f06-a228-4e4d-a1a8-3d5def999167
default.js:1496 Kernel: connected (6d8f9f06-a228-4e4d-a1a8-3d5def999167)
default.js:56 Starting WebSocket: ws://localhost:8888/api/kernels/087bbaf6-9342-4cc3-b4d1-7a26853762b7
default.js:1496 Kernel: connected (087bbaf6-9342-4cc3-b4d1-7a26853762b7)
renderer.js:45 Error: widget model not found
    at WidgetManager.get_model (manager.js:300)
    at WidgetRenderer.renderModel (renderer.js:38)
renderModel @ renderer.js:45
renderer.js:45 Error: widget model not found
    at WidgetManager.get_model (manager.js:300)
    at WidgetRenderer.renderModel (renderer.js:38)
renderModel @ renderer.js:45
renderer.js:45 Error: widget model not found
    at WidgetManager.get_model (manager.js:300)
    at WidgetRenderer.renderModel (renderer.js:38)
renderModel @ renderer.js:45
renderer.js:45 Error: widget model not found
    at WidgetManager.get_model (manager.js:300)
    at WidgetRenderer.renderModel (renderer.js:38)
renderModel @ renderer.js:45
renderer.js:45 Error: widget model not found
    at WidgetManager.get_model (manager.js:300)
    at WidgetRenderer.renderModel (renderer.js:38)
renderModel @ renderer.js:45
renderer.js:45 Error: widget model not found
    at WidgetManager.get_model (manager.js:300)
    at WidgetRenderer.renderModel (renderer.js:38)
renderModel @ renderer.js:45
default.js:56 Starting WebSocket: ws://localhost:8888/api/kernels/a258a6cc-666b-445c-8183-b081981c3113
default.js:1496 Kernel: connected (a258a6cc-666b-445c-8183-b081981c3113)
</pre>
</details>

--- a/packages/outputarea/style/base.css
+++ b/packages/outputarea/style/base.css
@@ -149,6 +149,12 @@ body.lm-mod-override-cursor .jp-OutputArea-output.jp-mod-isolated::before {
   display: none;
 }
 
+/* Hide empty lines in the output area, for instance due to cleared widgets */
+jp-OutputArea-prompt:empty {
+  padding: 0;
+  border: 0;
+}
+
 /*-----------------------------------------------------------------------------
 | executeResult is added to any Output-result for the display of the object
 | returned by a cell

--- a/packages/outputarea/style/base.css
+++ b/packages/outputarea/style/base.css
@@ -150,7 +150,7 @@ body.lm-mod-override-cursor .jp-OutputArea-output.jp-mod-isolated::before {
 }
 
 /* Hide empty lines in the output area, for instance due to cleared widgets */
-jp-OutputArea-prompt:empty {
+.jp-OutputArea-prompt:empty {
   padding: 0;
   border: 0;
 }

